src/phel/core.phel

```diff
@@ -677,6 +690,27 @@ arrays. Use (php/aunset ds key)" )))
     (php/aunset x key)
     x))
 
+(declare reduce)
```

Comment:
Is there a reason to avoid reduce or is it just a personal preference? Before I used 2 functions and foreach, but using reduce helped me making this function way shorter and more readable.

Reply:
Sorry, maybe I didn't made it clear. Using `reduce` is a good idea. I just wanted to move the set functions after the reduce function has been defined. Then you don't need the "declare" call. Does it make sense now?

Reply:
Oh ok, perfect then.